src/main/java/edu/ucsb/cs56/mapache_search/entities/AppUser.java

```diff
@@ -12,14 +12,25 @@
     @Id
     @GeneratedValue(strategy = GenerationType.IDENTITY)
     private long id;
+    private long searches; 
+    private final long maxsearches = 100l;  //100 queries per day for free: https://developers.google.com/custom-search/v1/overview
```

Comment:
Still not what I had in mind.

Consider:

```
public static final long MAX_SEARCHES=100l;
```

That way, the number 100l is stored only ONCE for the entire class.  There is no need to store that number in every object instance since it will be the same for each of them.  That's what static final fields are for.